pandas/_libs/lib.pyi

```diff
@@ -26,7 +27,7 @@ from enum import Enum
 class _NoDefault(Enum):
     no_default = ...
 
-no_default = _NoDefault.no_default
+no_default: Final = _NoDefault.no_default
```

Comment:
Needed for pyright see https://github.com/microsoft/pyright/issues/3546#issuecomment-1147644833